### DISCUSSION TITLE
Cannot find gas coin for signer address，这样算发布成功吗？

### DISCUSSION THREAD

**Discussion by ilses1** [hoh-zone/SuiStartrek#101] [Q&A] [answered]
发布合约是报以下错误：
![image](https://github.com/movefuns/SuiStartrek/assets/61105765/a9eb29e8-c62b-4199-b3e9-5006c3ba840a)
算发布成功吗，如果不成功，怎么解决问题啊

**Comment by ilses1** [comment, marked as answer]
该地址没有gas

解决方法：
1.从devnet或者testnet 获取gas
2.本地地址切换为对应的devnet或者testnet

**Comment by ilses1** [comment]
新的问题：有了对应地址的gas报错如下：

![image](https://github.com/movefuns/SuiStartrek/assets/61105765/042fda35-89d1-40cd-89d5-897263df09c9)

**Reply by jeffierw** [reply]
gas不够 多加个0
